src/main/java/org/fcrepo/camel/FcrepoHeaders.java

```diff
@@ -26,6 +26,8 @@
 
     public static final String FCREPO_TRANSFORM = "FCREPO_TRANSFORM";
 
+    public static final String HTTP_PREFER = "CamelHttpPrefer";
```

Comment:
Is there rationale for straying from the ALL_CAPS convention established by the previous three FCREPO headers?